credentials/src/main/kotlin/SSI.kt

```diff
@@ -228,6 +233,220 @@ public object VerifiableCredential {
     return JwtVerifiableCredential.fromCompactSerialization(vcJWT)
       .verify_Ed25519_EdDSA(publicKeyJWK.toOctetKeyPair())
   }
+
+  /**
+   * Converts a verifiable credential JWT into a [VerifiableCredentialType] object.
+   *
+   * @param vcJwt The verifiable credential in JWT format to be converted.
+   * @return A [VerifiableCredentialType] object representing the provided verifiable credential JWT.
+   * @throws Exception Throws an exception if the conversion from JWT to [VerifiableCredentialType] fails or any other unexpected error occurs during the conversion process.
+   */
+  @Throws(Exception::class)
+  public fun fromJwt(vcJwt: VcJwt): VerifiableCredentialType {
```

Comment:
Would be better off as an extension function IMHO. See https://kotlinlang.org/docs/coding-conventions.html#extension-functions

Reply:
Yup very cool, perfect use of this ✅